README.md

```diff
@@ -0,0 +1,252 @@
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- YOu have done a great job so far but there are few changes you need to make before moving to the next milestone. :+1: :+1: 

## Most of these changes were marked optional when your project got approved, but since this is a new review, you need to fix them before Proceeding.

- [ ] This logo is meant to be a place holder, You need to remove this or replace it with your own logo. :small_airplane:

Reply:
noted, removed lines of code